rotkehlchen/tests/unit/decoders/test_main.py

```diff
@@ -739,6 +739,7 @@ def test_reshuffle(ordered_events, events_list, result_list, msg='events_should_
     '0xA1E4380A3B1f749673E270229993eE55F35663b4',
     '0x756F45E3FA69347A9A973A725E3C98bC4db0b5a0',
 ]])
+@pytest.mark.parametrize('ethereum_manager_connect_at_start', ['DEFAULT'])
```

Comment:
I think we wll have to change this. I already had to change one test and re-record due to this. By giving `DEFAULT` as the nodes to use, the test's remote calls are no longer deterministic. If VCR is used, we can only use one node to connect per chain.